scribe/__init__.py

```diff
@@ -12,6 +14,16 @@
 app = Flask(__name__)
 app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///../data/temp.db'
 
+class JSONSecureCookieSession(SecureCookieSession):
+    serialization_method = json
+
+class JSONSecureCookieSessionInterface(SecureCookieSessionInterface):
+    session_class = JSONSecureCookieSession
+
+app.secret_key = 'foobar'
```

Comment:
Also idk how infosec best practices work but having the key in-line on github feels not so good. we could just file an issue and get around to making this better at a later point though.